WordPress/Classes/Utility/WPAssetExporter.h

```diff
@@ -1,6 +1,9 @@
 #import <Foundation/Foundation.h>
 #import <AssetsLibrary/AssetsLibrary.h>
 
+extern NSString * const WPAssetExportErrorDomain;
+extern const NSInteger WPAssetExportMissingAsset;
```

Comment:
Not very important but - maybe rename to `WPAssetExportErrorCodeMissingAsset`?  It's longer but it's also more immediately obvious.